src/database/seeds/0001-insert-base_fields.sql

```diff
@@ -17,283 +17,283 @@
-('Cause of Charity (Mid Level of cause categorization)' , 'cause_of_charity', '{ "type": "string" }' ),
-('Date of latest source' , 'date_of_latest_source', '{ "type": "string", "format": "date-time" }' ),
-('Is there a donor advisory on this charity?' , 'is_there_a_donor_advisory_on_this_charity', '{ "type": "boolean" }' ),
-('Text of the donor advisory message in html' , 'text_of_the_donor_advisory_message_in_html', '{ "type": "string" }' ),
-('Date the donor advisory went live' , 'date_the_donor_advisory_went_live', '{ "type": "string", "format": "date-time" }' ),
-('Date Current CEO was last verified' , 'date_current_ceo_was_last_verified', '{ "type": "string", "format": "date-time" }' ),
-('Web location of pro-forma 990s to handle Pharmaceutical valuation adjustments' , 'web_location_of_proforma_990s_to_handle_pharmaceutical_valuation_adjustments', '{ "type": "string" }' ),
-('Fiscal Year of Financial Rating' , 'fiscal_year_of_financial_rating', '{ "type": "string" }' ),
-('Fundraising Efficiency Amount' , 'fundraising_efficiency_amount', '{ "type": "integer" }' ),
-('Fundraising Expense Ratio' , 'fundraising_expense_ratio', '{ "type": "string" }' ),
-('Program Expense Ratio' , 'program_expense_ratio', '{ "type": "string" }' ),
-('Administrative Expense Ratio' , 'administrative_expense_ratio', '{ "type": "string" }' ),
-('Working Capital Ratio' , 'working_capital_ratio', '{ "type": "string" }' ),
-('Fundraising Efficiency Score' , 'fundraising_efficiency_score', '{ "type": "integer" }' ),
-('Fundraising Expense Score' , 'fundraising_expense_score', '{ "type": "integer" }' ),
-('Program Expense Score' , 'program_expense_score', '{ "type": "integer" }' ),
-('Administrative Expense Score' , 'administrative_expense_score', '{ "type": "integer" }' ),
-('Primary Revenue Growth Score' , 'primary_revenue_growth_score', '{ "type": "integer" }' ),
-('Program Expense Growth Score' , 'program_expense_growth_score', '{ "type": "integer" }' ),
-('Working Capital Score' , 'working_capital_score', '{ "type": "integer" }' ),
-('Score in the Financial dimension' , 'score_in_the_financial_dimension', '{ "type": "integer" }' ),
-('Score in the A&T dimension' , 'score_in_the_a&t_dimension', '{ "type": "integer" }' ),
-('Rating in the A&T dimension' , 'rating_in_the_a&t_dimension', '{ "type": "integer" }' ),
-('Score in the Results dimension' , 'score_in_the_results_dimension', '{ "type": "integer" }' ),
-('Rating in the Results Dimension' , 'rating_in_the_results_dimension', '{ "type": "integer" }' ),
-('Overall Score' , 'overall_score', '{ "type": "integer" }' ),
-('Overall Rating' , 'overall_rating', '{ "type": "integer" }' ),
-('Yes/No indicator: This rating is a Donor Advisory' , 'yes_no_indicator_this_rating_is_a_donor_advisory', '{ "type": "boolean" }' );
+INSERT INTO base_fields (label, description, short_code, data_type) VALUES
+('Organization Name', '', 'organization_name', '{ "type": "string" }' ),
```

Comment:
Just wanted to call out that I'm adding blank descriptions to the seed so that the seed file works against the updated schema.  (github collapses this file since so many lines were edited)

Reply:
What do you want to do about #305?